crates/router/src/connector/adyen/transformers.rs

```diff
@@ -292,6 +292,7 @@ impl ForeignTryFrom<(bool, AdyenWebhookStatus)> for storage_enums::AttemptStatus
             AdyenWebhookStatus::CancelFailed => Ok(Self::VoidFailed),
             AdyenWebhookStatus::Captured => Ok(Self::Charged),
             AdyenWebhookStatus::CaptureFailed => Ok(Self::CaptureFailed),
+            AdyenWebhookStatus::Reversed => Ok(Self::AutoRefunded),
```

Comment:
What does Reversed mean here, Is it a Voided payment? AutoRefunded occurs when we initiate a refund from Hyperswitch without merchant initiating it

Reply:
changed it, now it will through error for reversed in payment webhooks